exampleCourse/courseInstances/Sp15/assessments/plelementexamples/infoAssessment.json

```diff
@@ -1,45 +1,46 @@
 {
-    "uuid": "69aa799b-9492-4db2-9102-7753263f3863",
-    "type": "Homework",
-    "title": "PrairieLearn element example gallery",
-    "set": "Homework",
-    "number": "4",
-    "allowAccess": [
-        {
-            "mode": "Public",
-            "credit": 120,
-            "startDate": "2014-04-07T00:00:01",
-            "endDate": "2014-07-10T23:59:59"
-        },
-        {
-            "mode": "Public",
-            "credit": 100,
-            "startDate": "2014-07-07T00:00:01",
-            "endDate": "2024-07-10T23:59:59"
-        },
-        {
-            "mode": "Public",
-            "credit": 80,
-            "startDate": "2014-07-07T00:00:01",
-            "endDate": "2024-08-10T23:59:59"
-        },
-        {
-            "mode": "Public",
-            "credit": 50,
-            "startDate": "2014-07-07T00:00:01",
-            "endDate": "2024-09-10T23:59:59"
-        }
-    ],
-    "zones": [
-        {
-            "questions": [
-                {"id": "examplesCheckbox",           "points": 2, "maxPoints": 10},
-                {"id": "examplesStringInput",        "points": 2, "maxPoints": 10},
-                {"id": "elementsMatrixLatexDisplay", "points": 2, "maxPoints": 10},
-                {"id": "examplesMatrixInput",        "points": 2, "maxPoints": 10},
-                {"id": "examplesSymbolicInput",      "points": 2, "maxPoints": 10},
-                {"id": "examplesNumberInput",        "points": 2, "maxPoints": 10}
-            ]
-        }
-    ]
+  "uuid": "69aa799b-9492-4db2-9102-7753263f3863",
```

Comment:
This whole file had its indenting changed, which doesn’t seem necessary.

Reply:
Yeah, that was an accident, I auto-indented. Do you want me to try to revert it?